[1m============================= test session starts ==============================[0m
platform linux -- Python 3.10.12, pytest-6.2.5, py-1.10.0, pluggy-0.13.0
cachedir: /home/vboxuser/HW3_ws/build/py_pubsub/.pytest_cache
rootdir: /home/vboxuser/HW3_ws/src/py_pubsub
plugins: ament-copyright-0.12.9, launch-testing-1.0.4, ament-xmllint-0.12.9, ament-pep257-0.12.9, launch-testing-ros-0.19.6, ament-lint-0.12.9, ament-flake8-0.12.9, cov-3.0.0, colcon-core-0.15.1
[1mcollecting ... [0m[1m
collected 3 items                                                              [0m

test/test_copyright.py [33ms[0m[32m                                                 [ 33%][0m
test/test_flake8.py [31mF[0m[31m                                                    [ 66%][0m
test/test_pep257.py [32m.[0m[31m                                                    [100%][0m

=================================== FAILURES ===================================
[31m[1m_________________________________ test_flake8 __________________________________[0m
[1m[31mtest/test_flake8.py[0m:23: in test_flake8
    [94massert[39;49;00m rc == [94m0[39;49;00m, \
[1m[31mE   AssertionError: Found 1 code style errors / warnings:[0m
[1m[31mE     ./setup.py:24:68: W291 trailing whitespace[0m
[1m[31mE   assert 1 == 0[0m
----------------------------- Captured stdout call -----------------------------

./setup.py:24:68: W291 trailing whitespace
            'listener = py_pubsub.subscriber_member_function:main',         
                                                                   ^

1     W291 trailing whitespace

7 files checked
1 errors

'W'-type errors: 1

Checked files:

* ./setup.py
* ./py_pubsub/subscriber_member_function.py
* ./py_pubsub/__init__.py
* ./py_pubsub/publisher_member_function.py
* ./test/test_flake8.py
* ./test/test_copyright.py
* ./test/test_pep257.py
----------------------------- Captured stderr call -----------------------------
option --max-complexity: please update from optparse string `type=` to argparse callable `type=` -- this will be an error in the future
----- generated xml file: /home/vboxuser/HW3_ws/build/py_pubsub/pytest.xml -----
=========================== short test summary info ============================
FAILED test/test_flake8.py::test_flake8 - AssertionError: Found 1 code style ...
[31m============== [31m[1m1 failed[0m, [32m1 passed[0m, [33m1 skipped[0m, [33m2 warnings[0m[31m in 0.42s[0m[31m ==============[0m
